### PR TITLE
✨ Create Footer Button

### PR DESCRIPTION
**Pull request by mrana17**
- 4 Footer Button
- all styled
- include in storybook as story
- https://riding-heroe-footerbutt-v8pjtc.herokuapp.com/storybook/index.html?path=/story/common-footerbutton--history-button

### CODE REVIEW

File: components/footerButton/footerButton.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+export const HistoryButton = Template.bind({});
+HistoryButton.args = {
+  label: "History",
+  imgSrc: "/History.svg",
+  href: "#",
+};
+export const MapButton = Template.bind({});
+MapButton.args = {
+  label: "Map",
+  imgSrc: "route-map.svg",
```
Comment (cboenke) [inline components/footerButton/footerButton.stories.tsx]:
```suggestion
  imgSrc: "/route-map.svg",
```

File: components/footerButton/footerButton.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  href: "#",
+};
+export const ProfileButton = Template.bind({});
+ProfileButton.args = {
+  label: "Profile",
+  imgSrc: "Profile-picture.svg",
```
Comment (cboenke) [inline components/footerButton/footerButton.stories.tsx]:
```suggestion
  imgSrc: "/Profile-picture.svg",
```

File: components/footerButton/footerButton.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  href: "#",
+};
+export const WalletButton = Template.bind({});
+WalletButton.args = {
+  label: "Wallet",
+  imgSrc: "wallet.svg",
```
Comment (cboenke) [inline components/footerButton/footerButton.stories.tsx]:
```suggestion
  imgSrc: "/wallet.svg",
```

**Review by cboenke — CHANGES_REQUESTED** [review]
GJ! Add the slashes to the "imgSrc" so that all the images are shown and you could be consistent in your assets naming (e.g. all lower case).

**Review by DNaunin — APPROVED** [review]
LGTM! 🛵

**Review by phlgr — CHANGES_REQUESTED** [review]
Gj 🚀

**Review by RonjaMeinecke — APPROVED** [review]
🚀GJ

**Pull request merged**